anecdote/assets/logs/Anecdote Log Pipeline_test_tests.yaml

```diff
@@ -0,0 +1,63 @@
+id: anecdote
+tests:
+  - sample: >-
+      {
+      	"id": "AgAAAZB5YmFzhzsIeQAAAAAAAAAYAAAAAEFaQjVZbUZ6QUFEbVVJd21ZUWhyMVFDdQAAACQAAAAAMDE5MDc5NjItNjE3My00NjUzLWFiNTgtZTRkNWI3ZmE2ZjU0",
+      	"content": {
+      		"timestamp": "2024-07-03T16:16:49.011Z",
+      		"tags": [
+      			"is_first_bug_occurance:false",
+      			"app_version:-",
+      			"feedback_source:reddit",
+      			"source:anecdote",
+      			"is_feedback_bug:false",
+      			"datadog.submission_auth:api_key"
+      		],
+      		"service": "Anecdote",
+      		"message": "right. communism,  anarchism, not to mention fluffy rose colored everything should be free but no one should have to work?   \ni consider myself a centrist and imo this is the only position that actually looks at the issues.  the extremes on both sides are unrealistic and out to lunch.  both sides stake their claim to reality and *the truth* but are more similar than not.",
+      		"attributes": {
+      			"date": "2024-07-03 13:12:12",
+      			"service": "Anecdote",
+      			"tags": [
+      				"feedback_source:Reddit",
+      				"app_version:-",
+      				"is_feedback_bug:False",
+      				"is_first_bug_occurance:False"
+      			]
+      		}
+      	}
+      }
+    result: null
+  - sample: >-
+      {
+      	"id": "AgAAAZB5YmFzhzsIdgAAAAAAAAAYAAAAAEFaQjVZbUZ6QUFEbVVJd21ZUWhyMVFDcgAAACQAAAAAMDE5MDc5NjItNjE3My00NjUzLWFiNTgtZTRkNWI3ZmE2ZjU0",
+      	"content": {
+      		"timestamp": "2024-07-03T16:16:49.011Z",
+      		"tags": [
+      			"is_first_bug_occurance:false",
+      			"app_version:-",
+      			"feedback_source:reddit",
+      			"source:anecdote",
+      			"is_feedback_bug:false",
+      			"datadog.submission_auth:api_key"
+      		],
+      		"service": "Anecdote",
+      		"message": "why would you take an f if you didn't do it?",
+      		"attributes": {
+      			"date": "2024-07-03 08:46:56",
+      			"service": "Anecdote",
+      			"tags": [
+      				"feedback_source:Reddit",
+      				"app_version:-",
+      				"is_feedback_bug:False",
+      				"is_first_bug_occurance:False"
+      			]
+      		}
+      	}
+      }
+    result: null
+
+# The `result` field should be left blank to start. Once you submit your log asset files with
+# your integration pull-request in a Datadog GitHub repository, Datadog's validations will
+# run your raw logs against your pipeline and return the result. If the result output in the
+# validation is accurate, take the output and add it to the `result` field in your test YAML file.
```

Comment:
The validation has run and is now providing the result. Please copy the result from the validation and past it in the result field. You will need to repeat this for each sample. 

Ex:  https://github.com/DataDog/integrations-extras/pull/2347/files#annotation_24892241048